Quantum-Control-Applications/ESR/T1.py

```diff
@@ -0,0 +1,165 @@
+from qm import SimulationConfig
+from qm.qua import *
+from qm import LoopbackInterface
+from qm.QuantumMachinesManager import QuantumMachinesManager
+from configuration import *
+import matplotlib.pyplot as plt
+from qm.simulate.credentials import create_credentials
+
+################################
+# Open quantum machine manager #
+################################
+
+qmm = QuantumMachinesManager(host=qop_ip, port=qop_port)
+
+########################
+# Open quantum machine #
+########################
+
+qm = qmm.open_qm(config)
+
+###################
+# The QUA program #
+###################
+
+pi_len = int(320//4)
```

Comment:
no need for `int`. The `//` already keeps it as an int

Reply:
True. I have also noticed that when I do type(1e6//4) it returns me a float which is annoying when I do update frequency but I will save it for thoses cases

Reply:
For example, in the same code I have `cooldown_time = 10e6//4` and it is of type float but `wait()` requires type int

Reply:
Yes because 10e6 is a float, what we did in the other codes is to set qubit_T1 = int(10e6) and then declare cooldown_time = qubit_T1 // 4

Reply:
We need to fix `wait()` to accept a float :\ 
(duration accepts it)